react-redux/README.md

```diff
@@ -15,10 +15,15 @@ Please do the following **steps in this order**:
 ### Set-up ESlint in your local env - it will help you to find style errors
 1. run `npm install eslint eslint-config-airbnb --save-dev` (not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm))
 2. run `npx eslint --init`
-3. copy [.eslintrc.json](./.eslintrc.json) to the root directory of your project
-4. run `npx eslint .`
-5. fix linter errors
-6. **IMPORTANT NOTE**: feel free to research [auto-correct options for ESlint](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+3. make sure you select the following options when prompted
+
+![screenshot](../assets/images/eslint_config.png)
```

Comment:
Could you verify that this file is showing? Seems to be broken, also note that the file is `eslint_config.PNG` not `eslint_config.png`.

```suggestion
![screenshot](../assets/images/eslint_config.PNG)
```

Reply:
The path to the screenshot is going to the microverse repo that why it isn't showing, fixing it now

Reply:
Oh, I was viewing it from the commit that has the image: https://github.com/microverseinc/linters-config/blob/43f6e35499c9f4de04b16d0030fc9b6e35a66c71/react-redux/README.md

But it's all good now with the changes: https://github.com/microverseinc/linters-config/blob/ce946ae39b3a7f186e6533eb61e92c8ece3fc5bd/react-redux/README.md